symplyphysics/laws/astronomy/resolution_of_telescope.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+
+# Description
+## The resolution of the telescope is the minimum angular distance between point objects that can be
+## distinguished separately in the telescope.
+
+## Law is: r = 1.22 * (L / D), where
+## r - resolution of telescope,
+## L - wavelength,
+## D - lens diameter.
+
+resolution = Symbol("resolution", angle_type)
+
+wavelength = Symbol("wavelength", units.length)
+lens_diameter = Symbol("lens_diameter", units.length)
+
+constant_rad = Quantity(1.22 * units.radian)
```

Comment:
Do you have an idea, why `1.22`? If yes, add it to description.

Reply:
> Do you have an idea, why `1.22`? If yes, add it to description.

I don't know.